pkg/datarepair/checker/checker.go

```diff
@@ -36,14 +39,30 @@ func (c Config) Run(ctx context.Context, server *provider.Provider) (err error)
 
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
+	var errs []error
 
-	go func() {
+	go func() error {
 		for {
 			select {
 			case <-ticker.C:
 				zap.S().Info("Starting segment checker service")
+				go func() {
```

Comment:
we want to separate all the code from the config Run. We want the checker to have it's own run that config uses. This gives more flexibility on how we can start the checker later. see https://github.com/storj/storj/blob/master/pkg/datarepair/repairer/repairer.go#L50